src/components/FilmCard.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+      <b-card
+          :aria-hidden="show ? 'true' : null"
+          :title=title
+          :sub-title="originalTitle + ', ' + releaseDate.slice(0, 4)"
+          :img-src="backdropPath === null
+          ?'https://st4.depositphotos.com/17828278/24401/v/600/depositphotos_244011872-stock-illustration-image-vector-symbol-missing-available.jpg'
+          :'https://image.tmdb.org/t/p/original' + backdropPath"
+          footer-tag="footer"
+          img-alt="Image"
+          img-top
+          tag="article"
+          class="text-muted card-text form-control card-img-top b-card"
+      >
+        <b-card-text class="b-card__text">
+          {{ overview }}
+        </b-card-text>
+        <template #footer>
+          <div class="b-card__footer">
+            <b-form-rating readonly :value="overallRating" stars="10"></b-form-rating>
+          </div>
+        </template>
+      </b-card>
+</template>
+
+<script>
+export default {
+  name: "FilmCard",
+  data() {
+    return {
+      show: false
+    }
+  },
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+    backdropPath: {
+      type: String,
+      required: true
+    },
+    overview: {
+      type: String,
+      required: true,
+    },
+    originalLanguage: {
+      type: String,
+      required: true,
+    },
+    originalTitle: {
+      type: String,
+      required: true
+    },
+    releaseDate: {
+      type: String,
+      required: true
+    },
+    overallRating: {
+      type: Number,
+      required: true,
+    }
+  }
+}
+</script>
+
+<style scoped>
+  .text-muted {
+    color: #fff !important;
```

Comment:
I suppose that better choose will be delete this class (.text-muted) and you won't have to reassign styles for this tag. 
In other way it can be difficult to reassign styles in future.

Reply:
It's described this way because, without the !important keyword, I can't rewrite the style, because in the library itself, the color styles for the components are written this way, but in our case, this color is needed, and either one itself does not offer it, so I change it manually.